scripts/buildLocales.js

```diff
@@ -0,0 +1,31 @@
+import * as fs from 'fs'
+
+// This script loads the current JSON-based locales files and merges them into
+// a single importable ES module for bundling purposes
+const localesRoot = process.argv[2]
+if (!localesRoot) {
+    console.log('usage: buildLocales <locales/feature dir>')
+}
+
+const locales = {}
+const localesDirs = fs.readdirSync(localesRoot).filter(f => !f.startsWith('.'))
+for (const l of localesDirs) {
+    locales[l] = {}
+    const dir = `${localesRoot}/${l}`
```

Comment:
Probably better to use `path.join`, that takes care to get the separator right depending on platform. See https://nodejs.org/api/path.html#pathjoinpaths

(Same below.)

Reply:
Good call!